service/src/main/resources/logback.xml

```diff
@@ -6,9 +6,13 @@
 
   The console log pattern has been modified from the default to include request ID and user ID if present
   -->
-  <property name="CONSOLE_LOG_PATTERN" value="%clr(%d{${LOG_DATEFORMAT_PATTERN:-yyyy-MM-dd HH:mm:ss.SSS}}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint}%replace( %mdc - %msg){'  - ',' '}%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}" />
-  <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
-  <include resource="org/springframework/boot/logging/logback/console-appender.xml"/>
+  <property name="CONSOLE_LOG_PATTERN" value="%d{${LOG_DATEFORMAT_PATTERN:-yyyy-MM-dd HH:mm:ss.SSS}} ${LOG_LEVEL_PATTERN:-%5p} ${PID:- } --- [%15.15t] %-40.40logger{39} : %replace(%mdc - %msg){'  - ',' '}%n%ex" />
+
+  <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>${CONSOLE_LOG_PATTERN}</pattern>
+    </encoder>
+  </appender>
```

Comment:
It seems that the upgraded Spring boot version doesn't support some logging features and hence I:
- removed `clr()` function which was providing color formatting as it was throwing error `no conversion found`
- replaced `%wEx` with `%ex` which captures stack trace if any
- removed including `defaults.xml` and `console-appender.xml` as it was not exactly able to successfully include them (and throwing some conversion errors)
-  in this new Spring boot version it seem that you had to provide an appender in `logback.xml` and hence L11-15 were added. This is what made the tests print out human readable logging